code42_connector.py

```diff
@@ -154,11 +166,90 @@ def _handle_list_high_risk_employees(self, param, action_result):
         action_result.update_summary({"total_count": total_count})
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    """ALERTS ACTIONS"""
+
+    @action_handler_for("get_alert_details")
+    def _handle_get_alert_details(self, param, action_result):
+        alert_id = param["alert_id"]
+        response = self._client.alerts.get_details([alert_id])
+        alert = response.data["alerts"][0]
+        action_result.add_data(alert)
+        action_result.update_summary(
+            {"username": alert["actor"], "user_id": alert["actorId"]}
+        )
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    @action_handler_for("search_alerts")
+    def _handle_search_alerts(self, param, action_result):
+        username = param.get("username")
+        start_date = param.get("start_date")
+        end_date = param.get("end_date")
+        alert_state = param.get("alert_state")
+
+        if username is None and start_date is None and end_date is None and alert_state is None:
```

Comment:
Could be `if not username and not start_date and not end_date and not alert_state`, looks a little simpler and I don't think any of these params should work for falsy values

Reply:
could also do:

```
if not all(username, start_date, end_date, alert_state):
```